### ISSUE TITLE
When are emails sent?

### ISSUE THREAD

**Issue by glb** [canada-ca/devex#49]
I am subscribed to the [Create accessible copies of reports and summary tables opportunity](https://beta.gcdevexchange.org/opportunities/opp-create-accessible-copies-of-reports-and-summary-tables----cr-er-des-copies-accessibles-des-rapports-et-des-tableaux-sommaires) but haven't gotten any emails despite it going from open to closed and I think possibly other changes including description changes.

What events cause emails to be sent out? Is emailing known not to be working?

**Comment by RachelMuston** [comment]
Hi @glb sorry to hear you didn't receive any notification emails.  To confirm, when you say you are subscribed, does that mean you clicked on the 'follow' button at the top of the opportunity?

**Comment by glb** [comment]
Thanks @RachelMuston! Yes, I believe I did that some time ago. I did get an email saying that the opportunity had been updated this afternoon at 2:23PM Eastern. 

I haven't taken the time to go through the code here to find out when emails are expected to be sent, thought I would ask the question first since there was definitely an update. Maybe the previous update only happened on github?

**Comment by RachelMuston** [comment]
Thank you creating this issue @glb!  We found there was a problem with the notifications.  We fixed it this afternoon.  Based on the notifications I got, it looks like all the notifications that hadn't been sent due to the issue, got sent out once the fix was applied.  So that is why you got the notification this afternoon.

We also noticed that the notification messages contained two broken links ('See details' and 'unsubscribe') which we have also fixed but this will only apply to new notifications.

Thanks again for raising this!
